maas-api.yaml

```diff
@@ -1,555 +1,677 @@
           schema:
-            type: object
-            description: Use `modes` to filter by tranport modes. Only results for the given modes will be retuned. When `full` paramater is false, it will retrieve a list of route IDs for the given region/operator and `modes` and `onlyRealTime` if any. You can also retrieve information for a particular list of routes using `routeIDs` or `routeNames`, providing their IDs (or names) as they were given before as a list member in this very same endpoint. The use of filters like `modes`, `onlyRealTime`, `routeNames` and/or `routeIDs` could lead to a empty response. In order to prevent this, make sure such combinations are posible, using previously fetched information.
-            properties:
-              region:
-                type: string
-                description: Region name/code from `regions.json`.
-              operatorID:
-                type: string
-                description: TSP ID from `info/operator.json`.
-              modes:
-                type: array
-                items:
-                  type: string
-                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](https://developer.tripgo.com/faq/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
-              routesIDs:
-                type: array
-                items:
-                  type: string
-                  description: Route ID.
-                description: Route IDs to retrieve information from. In case of any missmatch, it'll return error response. Use either this filter or `routeNames`.
-              routesNames:
-                type: array
-                items:
-                  type: string
-                  description: Route name.
-                description: Route names to retrieve information from. In case of any missmatch, it'll return error response. Use either this filter or `routeIDs`.
-              onlyRealTime:
-                type: boolean
-                description: Boolean to filter only routes with Real Time support. When true, returned route list will contain only routes with Real Time service.
-                default: false
-              full:
-                type: boolean
-                description: Boolean if result should include full information for route/s or just a list with all their IDs.
-                default: false
-            required:
-              - region
-              - operatorID
-            example:
-              region: US_CA_LosAngeles
-              operatorID: LACMTA_Rail
-              modes:
-                - pt_pub_tram
-              full: true
+            type: string        
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RoutesResponse'
+
+  /agencies/{aid}/routes/{rid}/services:
```

Comment:
can we move this to `/services` and have parameters for `aid` and `rid`?

Reply:
Done